ios/Classes/OfflinePackDownloadManager.swift

```diff
@@ -56,7 +56,8 @@ class OfflinePackDownloader {
         let tilePyramidRegion = regionDefinition.toMGLTilePyramidOfflineRegion()
         storage
             .addPack(for: tilePyramidRegion,
-                     withContext: regionData.prepareContext()) { [weak self] pack, error in
+                     withContext: regionData.prepareContext())
```

Comment:
this file should be the cause of the format issues - this is a pure format change